addons/web/static/src/js/public/public_crash_manager.js

```diff
@@ -14,12 +14,11 @@ const PublicCrashManager = CrashManager.extend({
      * @override
      */
     _displayWarning(message, title, options) {
-        this.displayNotification({
-            ...options,
-            title,
-            message,
+        this.displayNotification(_.extend({
+            title: title,
```

Comment:
You can keep shorthand property names:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Browser_compatibility